pkg/pmem-csi-driver/main.go

```diff
@@ -48,6 +48,7 @@ func init() {
 
 	/* Controller mode options */
 	flag.StringVar(&config.schedulerListen, "schedulerListen", "", "controller: listen address (like :8000) for scheduler extender and mutating webhook, disabled by default")
+	flag.Var(&config.nodeSelector, "nodeSelector", "controller: reschedule PVCs with a selected node where PMEM-CSI is not meant to run because the node does not have these labels (represented as JSON map)")
```

Comment:
I am bit confused, where exactly the provided `nodeSelecter`  is used in the code, I could only see that this is used to check if the rescheduler should be started or not by the controller. But I couldn't find anywhere the contents of the provided nodeSeleter is used in the code. In this case why can't it be a simple boolean value. @pohly  Pardon and please correct me if I miss something.

Reply:
It's passed to the rescheduler here: https://github.com/intel/pmem-csi/blob/3a7d2cfc9483ad24c52c83a833865c02e158ea35/pkg/pmem-csi-driver/pmem-csi-driver.go#L219-L223

Reply:
Yes, but it is not used used by rescheduler?

Reply:
And it should be used here: https://github.com/intel/pmem-csi/blob/3a7d2cfc9483ad24c52c83a833865c02e158ea35/pkg/pmem-csi-driver/rescheduler.go#L196-L201

... but isn't! Duh. That code is of course bogus. Not sure what I was thinking. I'll fix it.

Reply:
I also need to add unit tests.

Reply:
Done, see updated commit.